home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1875,6 +1880,12 @@ components:
           example: "77778"
           minLength: 1
           maxLength: 50
+        admin_email:
```

Comment:
Why do we need it? I think this information must be hidden until admin creates a contact and expose the email address explicitly. See `contacts` property.

Reply:
Thanks, forgot to delete